src/Notifications/notificationTabs.test.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+import {
+  act, fireEvent, render, screen, within,
+} from '@testing-library/react';
+import { Context as ResponsiveContext } from 'react-responsive';
+import { Factory } from 'rosie';
+
+import { initializeMockApp } from '@edx/frontend-platform';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { AppContext, AppProvider } from '@edx/frontend-platform/react';
+
+import AuthenticatedUserDropdown from '../learning-header/AuthenticatedUserDropdown';
+import { initializeStore } from '../store';
+import setupLearnerMockResponse from './test-utils';
+
+import './data/__factories__';
+
+let store;
+
+async function renderComponent() {
+  await render(
+    <ResponsiveContext.Provider>
+      <IntlProvider locale="en" messages={{}}>
+        <AppProvider store={store}>
+          <AppContext.Provider>
+            <AuthenticatedUserDropdown />
+          </AppContext.Provider>
+        </AppProvider>
+      </IntlProvider>
+    </ResponsiveContext.Provider>,
+  );
+}
+
+describe('Notification Tabs test cases.', () => {
+  beforeEach(async () => {
+    initializeMockApp({
+      authenticatedUser: {
+        userId: '123abc',
+        username: 'testuser',
+        administrator: false,
+        roles: [],
+      },
+    });
+
+    Factory.resetAll();
+    store = initializeStore();
+
+    ({ store } = await setupLearnerMockResponse());
```

Comment:
Test cases are related to notifications. Please update the function name. 
`mockNotificationsResponse`